app/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,13 @@
   <string name="topic_overview_see_more">See More</string>
   <string name="topic_download_text">(%s MB)</string>
   <string name="topic_story_progress_percentage">%s\%%</string>
+  <string name="continue_lesson">Continue with your Lesson</string>
+  <string name="more">MORE</string>
```

Comment:
We should capitalize using textAllCaps and not here.

Reply:
Actually this string was not getting used anywhere so removed it.